toolkit-test-app/src/main/java/com/esri/arcgisruntime/toolkit/test/sceneview/ArcGISSceneViewTableTopActivity.kt

```diff
@@ -0,0 +1,154 @@
+                        surface.elevationSources.add(it)
+                    }
+                    surface.navigationConstraint = NavigationConstraint.NONE
+                    surface.opacity = 0f
+                    scene.baseSurface = surface
+                    val encodedString =
+                        URI.create("https://tiles.arcgis.com/tiles/OLiydejKCZTGhvWg/arcgis/rest/services/3D_Punktwolke_Dome_Köln/SceneServer/layers/0")
+                    pointCloudLayer = PointCloudLayer(encodedString.toASCIIString()).also {
+                        it.addLoadStatusChangedListener { loadStatusChangedEvent ->
+                            if (loadStatusChangedEvent.newLoadStatus == LoadStatus.LOADED) {
+                                showSnackbar(R.string.arcgis_sceneview_tabletop_activity_loading_point_cloud_layer_complete)
+                            } else if (loadStatusChangedEvent.newLoadStatus == LoadStatus.FAILED_TO_LOAD) {
+                                Toast.makeText(
+                                    this@ArcGISSceneViewTableTopActivity,
+                                    getString(
+                                        R.string.arcgis_sceneview_tabletop_activity_loading_point_cloud_layer_failed,
+                                        "${it.loadError.message} - ${it.loadError.cause}"
+                                    ),
+                                    Toast.LENGTH_LONG
+                                ).show()
+                            }
+                        }
+                        it.loadAsync()
+                        showSnackbar(R.string.arcgis_sceneview_tabletop_activity_loading_point_cloud_layer)
+                    }
+                }
+            }
+            ArcGISArView.ArcGISArViewState.INITIALIZATION_FAILURE -> {
+                with(getString(R.string.arcgisarview_error, arcGisArView.error?.message)) {
+                    Log.e(logTag, this)
+                    Toast.makeText(this@ArcGISSceneViewTableTopActivity, this, Toast.LENGTH_LONG).show()
+                }
+            }
+        }
+    }
+
+    private fun showSnackbar(@StringRes stringRes: Int) {
+        Snackbar.make(arcGisArView, stringRes, Snackbar.LENGTH_LONG).show()
+    }
+
+    override fun onCreateOptionsMenu(menu: Menu?): Boolean {
+        menuInflater.inflate(R.menu.arcgissceneview_tabletop_activity, menu)
+        return true
+    }
+
+    override fun onOptionsItemSelected(item: MenuItem?): Boolean {
+        when (item?.itemId) {
+            R.id.action_start_tracking -> {
+                arcGisArView.startTracking()
+                return true
+            }
+            R.id.action_stop_tracking -> {
+                arcGisArView.stopTracking()
+                return true
+            }
+        }
+        return super.onOptionsItemSelected(item)
+    }
+
+    override fun onDestroy() {
+        arcGisArView.removeOnStateChangedListener(this)
```

Comment:
keep an out-commented line here to destroy the ArcGISArView?

Reply:
Because it's a lifecycle aware component, it's destroyed when the `Activity` is without me manually calling it:
```
arcGisArView.registerLifecycle(lifecycle)
```

Reply:
Of course, thanks!